tests/system/robot/chromeTests.py

```diff
@@ -157,7 +158,8 @@ def test_mark_aria_details_role():
 				<div id="endnote-details" role="doc-endnote">details with role doc-endnote</div>
 				<div id="footnote-details" role="doc-footnote">details with role doc-footnote</div>
 				<div id="comment-details" role="comment">details with role comment</div>
-				<div id="definition-details">details with role definition</div>
+				<div id="definition-details-as-role" role="definition">details with role definition</div>
+				<definition id="definition-details-as-tag">details with tag definition</definition>

```

Comment:
I don't think this is right. See:
- https://codepen.io/reefturner/pen/PoQXKeJ
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dfn
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/definition_role

Reply:
I have removed the use of `<dfn>`.
`<dfn>` is used for the term of the definition, and maps to the text of the parent node.
I don't believe this syntax is useful / supported by `aria-details`.

```html
<p><dfn>Mansplaining</dfn>, a portmanteau of "man" and "explain", is the patronizing act of explaining without being asked to do so, to someone already learned on the topic, often after someone has already explained it. </p>
```

Reply:
MDN suggests that the `dfn` elements should be preferred over the alternative approach. Even if unsupported, I think it worth having in the test.

Web devs will likely expect to be able to the following and get "has definition". Although still unsupported, recording the use-case for future consideration will help to prevent missing it later.
```HTML
<div>
    <p>Some topic that may be related to <span aria-details="def1">mansplaining</span>.</p>
</div>
<div>
    <p><dfn id="def1">Mansplaining</dfn>, a portmanteau of "man" and "explain", is the patronizing act of explaining</p>
</div>
```

That said, I agree this looks difficult to support, there seems to be a disconnect between term vs definition for the term.

Reply:
I don't believe this syntax would be appropriate, as the developer would be intending to target the definition of the term, which is the unlabelled `<p>` tag.